Revit_Core_Engine/Query/ViewSolid.cs

```diff
@@ -115,6 +165,12 @@ private static Line ViewDepthLine(this View view)
             }
         }
 
+        /***************************************************/
+        /****              Private fields               ****/
+        /***************************************************/
+
+        private static double m_DefaultHorizontalExtents = 1e+6;
```

Comment:
Could we please put this field next to `m_DefaultExtents` and rename the latter to `m_DefaultVerticalExtents`?

Reply:
Done 👍